### DISCUSSION TITLE
Unable to use my own ether(pad)(calc) instance

### DISCUSSION THREAD

**Discussion by bitsandnumbers** [otetard/ownpad#125] [Q&A]
Hi, and thanks for this great plugin !

I'm trying to use ownpad with my own EtherCalc and EtherPad instances, but when trying to open a .pad or .calc file I get this error : "Your Etherpad/Ethercalc document could not be opened: the URL specified in your file does not match the allowed server". Everything work fine if I use the official pad/calc instances. Any idea on how to allow my own servers ? thx, Jos

**Comment by mr-bolle** [comment]
Hi, 

do you use your own Etherpad-Lite Instance? 
Or you enter only your Hostdomain as Etherpad Host or Ethercalc Host?

You can try the public Etherpad-Lite Collaboration Server -->[Wiki with several Server](https://github.com/ether/etherpad-lite/wiki/Sites-that-run-Etherpad-Lite)

Currently i would create a own etherpad-lite note server with [etherpad-docker](https://github.com/fuerst/etherpad-docker/tree/stable) but I don't get currently a working solution.

**Comment by bitsandnumbers** [comment]
Hi @mr-bolle , I have a etherpad-docker instance of my own located on a subdomain of the same server. So I enter this subdomain in ownpad settings.

Right now, I'm using the public Etherpad-Lite Collaboration Server. I'd like to use the etherpad-docker instance instead, but I get the error above :/

**Comment by TheOneWithTheBraid** [comment]
Same issue here: I am unable to connect to a docker instance of Etherpad. Maybe due to CORS?
